shared/actions/teams.tsx

```diff
@@ -1334,15 +1362,18 @@ const badgeAppForTeams = (state, action) => {
     // Covers case where we have a badge appear on the requests
     // tab with no rows showing up
     const newTeamRequestsSet = I.Set(newTeamRequests)
+    // TODO ts-migration remove any
     const existingNewTeamRequestsSet = I.Set(existingNewTeamRequests)
-    const toLoad = newTeamRequestsSet.subtract(existingNewTeamRequestsSet)
+    // TODO ts-migration remove any
+    const toLoad: I.Set<any> = newTeamRequestsSet.subtract(existingNewTeamRequestsSet)
     const loadingCalls = toLoad.map(teamname => TeamsGen.createGetDetails({teamname})).toArray()
     actions = actions.concat(loadingCalls)
   }
 
   // if the user wasn't on the teams tab, loads will be triggered by navigation around the app
   actions.push(
     TeamsGen.createSetNewTeamInfo({
+      // @ts-ignore a legit error. todo fix
```

Comment:
@cjb I think this is a legit error from your PR. You forgot to add this in the types